crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -838,16 +839,23 @@ def upgrade_controller_clusters
 
         return upgrade_controllers_disruptive if upgrade_mode == :normal
 
-        network_node = ::Node.find(
-          "pacemaker_founder:true AND " \
+        network_founders = ::Node.find(
+          "pacemaker_founder:* AND " \
           "run_list_map:neutron-network AND NOT " \
           "run_list_map:neutron-server"
-        ).first
+        )
+        network_node = ::Node.load(network_founders.first[:pacemaker][:founder]) rescue nil
```

Comment:
Is the `rescue nil` needed here? Won't the load return nil anyway, if the node does not exist?

Reply:
Well, it cannot hurt I guess